apps/checker/cmd/main.go

```diff
@@ -136,6 +193,8 @@ func main() {
 				Timestamp:     req.CronTimestamp,
 				MonitorID:     req.MonitorID,
 				WorkspaceID:   req.WorkspaceID,
+				Error:         1,
```

Comment:
Are you sure about that?

Reply:
Yep this happens when we have an error at the third retry